mtp_api/apps/core/management/commands/load_test_data.py

```diff
@@ -1,22 +1,91 @@
-from django.core.management.base import BaseCommand
-from django.core.management import call_command
-from django.contrib.auth.models import User
+import textwrap
 
-from core.tests.utils import make_test_users
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.core.management.base import NoArgsCommand
 
+from core.tests.utils import make_test_users, give_superusers_full_access
+from prison.models import Prison
+from transaction.models import Transaction
 from transaction.tests.utils import generate_transactions
 
 
-class Command(BaseCommand):
+class Command(NoArgsCommand):
```

Comment:
According to docs, NoArgsCommand is deprecated in 1.8: https://docs.djangoproject.com/en/1.9/howto/custom-management-commands/#django.core.management.NoArgsCommand